cli/cmd/initnodes.go

```diff
@@ -398,3 +423,96 @@ func gethInit(ctx context.Context, cfg InitConfig, dir string) error {
 
 	return nil
 }
+
+func restoreSnapshotData(ctx context.Context, homeDir string, clientName string) error {
+	log.Info(ctx, fmt.Sprintf("⌛ Restoring latest chain snapshot for %s...", clientName))
+
+	clientStorageDir, err := getClientStorageDir(clientName)
+	if err != nil {
+		return err
+	}
+
+	absClientStorageDir := filepath.Join(homeDir, clientStorageDir)
+	origAbsClientStorageDir := fmt.Sprintf("%s_orig_%d", absClientStorageDir, time.Now().Unix())
```

Comment:
lets keep things simple, no need for backups. Just error if dir not empty.

Reply:
The whole idea was to keep any preexisting state or other modification that the user might have had until this point. But if that is a non issue, I will just not back up the original configs.

Reply:
always keep v1 as simple as possible

Reply:
Done.